backend/app/services/knowledge/permission_service.py

```
@@ -0,0 +1,363 @@
+    Raises:
+        ValueError: If user is not admin and KB doesn't exist
+    """
+    from datetime import datetime
+
+    # Look for existing organization KB
+    existing = (
+        db.query(Kind)
+        .filter(
+            Kind.kind == "KnowledgeBase",
+            Kind.namespace == "organization",
+            Kind.is_active == True,
```

Ruff (0.14.14)

Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)